### DISCUSSION TITLE
AttributeError: module 'arff' has no attribute 'COO'

### DISCUSSION THREAD

**Discussion by LSL-ding** [scikit-multilearn/scikit-multilearn#276] [Q&A] [answered]
I use load_from_arff methon in skmultilearn to read 'enron.arff' file,but It reports that "AttributeError: module 'arff' has no attribute 'COO'".I've tried to use earlier version of liac-arff , arff and ,but the error persists.my python version is 3.5.6,and arff version is 0.9,liac-arff version is 2.2.0, skmultilearn version is 0.1.0 now 
thanks for any answer and advice

**Comment by lxd2** [comment, marked as answer]
Make sure that you install liac-arrf instead of arff package
